README.md

```diff
@@ -50,6 +50,28 @@ have dependencies on other algebras which must be implemented.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## Type signature notation
+
+The type signature notation used in this document is described [here](https://sanctuary.js.org/#types). In summary:
+
+1. New types can be created via type constructors. Type constructors take one or more type arguments. For example, the type "array of strings" is written as `Array String`, and the type constructor `Array` can be defined as `Array :: a -> a`.
```

Comment:
@davidchambers I've updated [this section in the README.md](https://github.com/justin-calleja/fantasy-land/tree/issue-222#type-signature-notation). Basically, my current understanding is there so feel free to slice and dice it as necessary 😄 

I think I get it though. A lot hinges on properly understanding `Type`. I have defined it as the type of a "type expression" in the README.md (based on some looking around). Any thoughts on this?